elasticgraph-schema_definition/lib/elastic_graph/schema_definition/results.rb

```diff
@@ -300,6 +310,10 @@ def identify_extra_update_targets_by_object_type_name
             full_errors << "Schema had #{relationship_errors.size} error(s) related to relationship fields:\n\n#{relationship_errors.map.with_index(1) { |e, i| "#{i}. #{e}" }.join("\n\n")}"
           end
 
+          if has_had_multiple_sources_errors.any?
+            full_errors << "Schema had #{has_had_multiple_sources_errors.size} error(s) related to `has_had_multiple_sources!`:\n\n#{has_had_multiple_sources_errors.map.with_index(1) { |e, i| "#{i}. #{e}" }.join("\n\n")}"
```

Comment:
I think it may be confusing to tell the user that their schema has an error related to `has_had_multiple_sources!` when their schema doesn't call `has_had_multiple_sources!` anywhere and they probably don't even know it exists.  I think we can just include the new error in `sourced_field_errors` (rather than tracking `has_had_multiple_sources_errors` separately) so that it is reported as being related to `sourced_from` -- which it is.